### PR TITLE
Simplify Test Pages

### PR DESCRIPTION
**Pull request by seanpdoyle**
Dedicate each feature test configuration its own [HighVoltage] route and
view.

Decreases clutter in test pages and allows new tests to occupy their own
view.

Additionally, we must modify the `my-app` configuration to use the
`hash` router, so that our JS won't fail depending on the URL of the
route being loaded.

[HighVoltage]: https://github.com/thoughtbot/high_voltage

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ gem "rails", "4.2.4"
 gem "pry"
 
 group :development, :test do
+  gem "high_voltage", "~> 2.4.0"
```
Comment (geoffharcourt) [inline Gemfile]: This is an extremely elegant approach. :+1:

File: bin/setup [hunk 2]
```diff
@@ -1,7 +1,11 @@
-#!/bin/sh
+#!/usr/bin/env bash
```
Comment (geoffharcourt) [inline bin/setup]: Was there something not `sh`-compatible that drove this change?

**Comment by geoffharcourt** [comment]
You've already merged, but looks good. One question about the hashbang change.

File: bin/setup (hunk 2 above)
Comment (seanpdoyle) [inline bin/setup]: It complained about `function()`

**Pull request merged**